protocol/contracts/oracle/HybridOraclePool.sol

```diff
@@ -0,0 +1,160 @@
+
```

Comment:
This contract takes the place of the old `Oracle`, and sits as an aggregator in from of a suite of Uniswap Pairs. 
It conforms to the same capture() interface as the old Oracle, so from the DAO's point of view it is still looking at a single source of truth.  We can add and remove new pairs, and the weight of the TWAP contained those pairs is determined by the USD value of the LP tokens bonded within the accompanying HybridPool for a given HybridOracle.